lib/awis/models/url_info.rb

```diff
@@ -33,7 +33,7 @@ def setup_data!(response)
 
         response.each_node do |node, path|
           text = node.inner_xml
-          text = text.nil? || text.empty? ? nil : text.to_i if text.nil? || text.empty? ? nil : text.to_i.to_s == text && node.name != 'aws:Delta'
+          text = text.to_i if text.to_i.to_s == text && node.name != 'aws:Delta'
```

Comment:
I missed the connection of this logic to the changes around `text` many lines below.  Need to work on this more.